lib/get-class-name.test.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+var assert = require("@sinonjs/referee").assert;
+var getClassName = require("./get-class-name");
+
+describe("getClassName", function() {
+    it("returns the class name of an instance", function() {
+        // Because eslint-config-sinon disables es6, we can't
+        // use a class definition here
+        // https://github.com/sinonjs/eslint-config-sinon/blob/master/index.js
+        // var instance = new (class TestClass {})();
+        var instance = new function TestClass() {}();
+        var name = getClassName(instance);
+        assert.equals(name, "TestClass");
+    });
+
+    it("returns the Object for {}", function() {
```

Comment:
I think it would read better if it was clear that the expected return value is a string. Perhaps something like:

`returns 'Object' for {}`

Reply:
Makes sense, thanks